collection/sites/clyp.xml

```diff
@@ -0,0 +1,9 @@
+<site name="Clyp" homepage="https://clyp.it/">
+	<example>https://clyp.it/0r0qbxx2</example>
+
+	<tags><tag>music</tag></tags>
+
+	<host>clyp.it</host>
+	<extract>!//clyp.it/(?'id'[\w]+)!</extract>
+	<iframe src="https://clyp.it/{@id}/widget"/>
```

Comment:
`<iframe width="100%" height="265" src="https://clyp.it/{@id}/widget" frameborder="0"/>`